package.json

```diff
@@ -19,16 +19,16 @@
   },
   "dependencies": {
     "async": "~2.6.3",
-    "chalk": "~1.1.3",
     "cheerio": "~1.0.0-rc.3",
     "commander": "~2.20.3",
     "globby": "~6.1.0",
+    "kleur": "~3.0.3",
     "lodash": "~4.17.20",
     "node-fetch": "~2.6.0",
     "pa11y": "~5.3.0",
+    "pa11y-reporter-cli": "~2.0.0",
```

Comment:
Question for @pa11y/pa11y-reviewers, any idea why `pa11y-ci` didn't use the reporter before? 

*for the record, I think using the reporter here is a great idea!